src/wp-includes/theme-templates.php

```diff
@@ -1,5 +1,33 @@
 <?php
 
+/**
+ * Sets a custom slug when creating auto-draft template parts.
+ * This is only needed for auto-drafts created by the regular WP editor.
+ * If this page is to be removed, this won't be necessary.
+ *
+ * @param int $post_id Post ID.
```

Comment:
```suggestion
 * Sets a custom slug when creating auto-draft template parts.
 *
 * This is only needed for auto-drafts created by the regular WP editor.
 * If this page is to be removed, this won't be necessary.
 *
 * @since 5.9.0
 *
```